### PR TITLE
fail gate tests

### PR DESCRIPTION
**Pull request by gandelman-a**

### CODE REVIEW

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.org/check_github/BonnyCI/sandbox/24/1492449435.45/bonnyci-run-check : FAILURE in 42s

File: .bonnyci/run.sh [hunk 1]
```diff
@@ -3,4 +3,10 @@
 echo $BONNYCI_TEST_PIPELINE > $BONNYCI_TEST_LOG_DIR/pipeline
 dmesg >$BONNYCI_TEST_LOG_DIR/dmesg
 sleep 30
-echo "True"
\ No newline at end of file
+echo "True"
+
+if [[ "$BONNYCI_TEST_PIPELINE" == "gate" ]]; then
+  exit 1
+elif
```
Comment (ConsoleCatzirl) [inline .bonnyci/run.sh]: This should just be an 'else'

**Review by ConsoleCatzirl — DISMISSED** [review]

**Review by SpamapS — APPROVED** [review]

**Comment by gandelman-a** [comment]
gate-recheck

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.org/gate_github/BonnyCI/sandbox/24/1492449889.77/bonnyci-run-gate : FAILURE in 42s

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.org/gate_github/BonnyCI/sandbox/24/1492449936.61/bonnyci-run-gate : FAILURE in 42s

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.org/gate_github/BonnyCI/sandbox/24/1492449983.61/bonnyci-run-gate : FAILURE in 39s

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.org/gate_github/BonnyCI/sandbox/24/1492450036.53/bonnyci-run-gate : FAILURE in 42s

**Comment by gandelman-a** [comment]
gate-recheck

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.org/gate_github/BonnyCI/sandbox/24/1492450397.79/bonnyci-run-gate : FAILURE in 42s

**Comment by gandelman-a** [comment]
gate-recheck

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.org/gate_github/BonnyCI/sandbox/24/1492450466.89/bonnyci-run-gate : FAILURE in 42s

**Pull request closed without merging**